app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= render partial: 'partials/navigation_menu', locals: {visited: 'recipes'} %>
+<main class="show-recipe dflex-col">
+  <h2><%= @recipe.name %></h2>
+  <div class="dflex-row line1">
+    <p><span>Preparation Time:</span> <%= @recipe.preparation_time %>hour</p>
+    <%= render partial: 'switch_button', locals: {public: @recipe.public} %>
+  </div>
+  <div class="dflex-col line2">
+    <p><span>Cooking time:</span> <%= @recipe.cooking_time %>hours</p>
+    <p><span>Description:</span> <br/> <%= @recipe.description %></p>
+  </div>
+
+  <div class="dflex-row line3">
+    <%= button_to 'Generate Shopping List', '/general_shopping_list', method: :get %>
+    <% if can? :add_new_food, @recipe %>
+      <%= button_to 'Add Ingredient', '/foods/new', method: :get %>
+    <% end %>
```

Comment:
- [ ] This feature mentioned by the first reviewer hasn't been implemented yet! Please change the link_path in the button element to take the user to the form where they can add new ingredients to the current showed recipe. 🙏